dds/src/implementation/rtps_udp_psm/mapping_rtps_messages/submessage_elements/entityid.rs

```diff
@@ -1,14 +1,38 @@
-use std::io::{Error, Write};
+use std::{
+    convert::TryInto,
+    io::{Error, Write},
+};
 
 use byteorder::ByteOrder;
 
 use crate::implementation::{
-    rtps::messages::submessage_elements::EntityIdSubmessageElement,
+    rtps::{messages::submessage_elements::EntityIdSubmessageElement, types::EntityId},
     rtps_udp_psm::mapping_traits::{
         MappingReadByteOrdered, MappingWriteByteOrdered, NumberOfBytes,
     },
 };
 
+impl MappingWriteByteOrdered for EntityId {

```

Comment:
I think I would prefer if the MappingWriteByteOrdered  and MappingReadByteOrdered traits would be implemented only for the SubmessageElement types since those are the ones that are meant to be mapped onto the wire.